packages/cli-kit/src/ui/executor.ts

```diff
@@ -1,33 +1,23 @@
-import {AutoComplete} from './enquirer/autocomplete.js'
-import {Input} from './enquirer/input.js'
-import {Select} from './enquirer/select.js'
 import {CustomInput} from './inquirer/input.js'
 import {CustomAutocomplete} from './inquirer/autocomplete.js'
 import {CustomSelect} from './inquirer/select.js'
 import {CustomPassword} from './inquirer/password.js'
-import {isTruthy} from '../environment/utilities.js'
 import {Question} from '../ui.js'
 import inquirer, {Answers, QuestionCollection} from 'inquirer'
-import enquirer from 'enquirer'
 import fuzzy from 'fuzzy'
 
 export function getMapper(): (question: Question) => unknown {
-  return isEnquirer() ? enquirerMapper : inquirerMapper
+  return inquirerMapper
```

Comment:
What about moving the code of inquirerMapper directly here?

Reply:
This `executor` was originally planned as an abstraction layer for the cli ui module. The `executor` implements integration with different external libraries, so adding a new one doesn't require to modify cli core code base. Initially enquirer and inquirer were supported. We could remove completely the `executor` implementation for now